sheetstorm/tests/test_integrity_generate_test_data.py

```diff
@@ -0,0 +1,228 @@
+        self.assertEqual(
+            Project.objects.filter_completed().count(), number_of_projects_to_create[ProjectType.COMPLETED.name]
+        )
+
+    def test_command_should_not_create_any_project_if_there_is_no_specified_any_argument(self):
+        management.call_command("generate_test_data")
+
+        self.assertEqual(Project.objects.all().count(), 0)
+
+    @parameterized.expand(
+        [(ProjectType.SUSPENDED.name, 2), (ProjectType.ACTIVE.name, 2), (ProjectType.COMPLETED.name, 2)]
+    )
+    def test_command_should_not_require_all_arguments_to_create_specified_projects(
+        self, project_type, number_of_projects_to_create
+    ):
+        management.call_command("generate_test_data", **{project_type: number_of_projects_to_create})
+
+        self.check_number_of_projects_in_database(project_type, number_of_projects_to_create)
+
+    @parameterized.expand(
+        [
+            (ProjectType.SUSPENDED.name, 0, 0),
+            (ProjectType.SUSPENDED.name, -100, 0),
+            (ProjectType.SUSPENDED.name, 2, 2),
+            (ProjectType.ACTIVE.name, 0, 0),
+            (ProjectType.ACTIVE.name, -100, 0),
+            (ProjectType.ACTIVE.name, 2, 2),
+            (ProjectType.COMPLETED.name, 0, 0),
+            (ProjectType.COMPLETED.name, -100, 0),
+            (ProjectType.COMPLETED.name, 2, 2),
+        ]
+    )
+    def test_command_should_not_create_projects_if_specified_number_is_not_greater_than_0(
+        self, project_type, number_of_projects_to_create, expected_number_of_projects_in_database
+    ):
+        management.call_command("generate_test_data", **{project_type: number_of_projects_to_create})
+
+        self.check_number_of_projects_in_database(project_type, expected_number_of_projects_in_database)
+
+    @parameterized.expand(
+        [(ProjectType.SUSPENDED.name, 1), (ProjectType.ACTIVE.name, 1), (ProjectType.COMPLETED.name, 1)]
+    )
+    def test_despite_filled_database_command_should_create_specified_number_of_projects(
+        self, project_type, number_of_projects
+    ):
+        management.call_command("generate_test_data", **{project_type: number_of_projects})
+
+        management.call_command("generate_test_data", **{project_type: number_of_projects + 1})
+
+        self.check_number_of_projects_in_database(project_type, number_of_projects + 1)
+
+    def check_number_of_projects_in_database(self, project_type, expected_number_of_projects):
+        if project_type == ProjectType.SUSPENDED.name:
+            self.assertEqual(Project.objects.filter_suspended().count(), expected_number_of_projects)
+        elif project_type == ProjectType.ACTIVE.name:
+            self.assertEqual(Project.objects.filter_active().count(), expected_number_of_projects)
+        elif project_type == ProjectType.COMPLETED.name:
+            self.assertEqual(Project.objects.filter_completed().count(), expected_number_of_projects)
+
+
+class CombinedOptionsTests(TestCase):
```

Comment:
There are few tests which I think should be added. Try to call command with:
- "--admin 'test-text'",
- "--employee 'test-text'",
- "--manager 'test-text'",
- "--superuser 1",
- "--superuser 'test-text'",
etc. 

You can do this using parameterized tests ;)